packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -83,15 +83,15 @@ export interface BaseServiceOptions {
    *
    * Valid values are: PropagateTagFromType.SERVICE, PropagateTagFromType.TASK_DEFINITION or PropagateTagFromType.NONE
```

Comment:
should `PropagateTagFromType` be `PropagatedTagSource`

Reply:
+1

Reply:
Good catch!